app/views/manuals/_header.html.erb

```diff
@@ -7,15 +7,20 @@
       <span class='manual-type'>HMRC internal manual</span>
     <% end %>
 
-    <h1 id="manual-title"><%= @manual.title %></h1>
-    <% if @manual.organisations.any? %>
-      <dl>
-        <dt>From</dt>
-        <dd><%= links_to_sentence(@manual.organisations) %></dd>
-        <dt>Updated</dt>
-        <dd><%= marked_up_date(@manual.updated_at) %>, <%= link_to 'see all updates', @manual.updates_url %></dd>
-      </dl>
-    <% end %>
+    <div id="manual-title"><%= render partial: 'govuk_component/title', locals: {
+      title: @manual.title,
+      average_title_length: "long"
+    } %></div>
+    <%= render partial: 'govuk_component/metadata', locals: {
+      from: @manual.organisations.map { |org|
```

Comment:
Could some of this logic move into the presenter?

In government-frontend the parameters for the component are managed in the presenter, keeping the template simple and re-usable: eg https://github.com/alphagov/government-frontend/blob/master/app/presenters/metadata.rb#L5 https://github.com/alphagov/government-frontend/blob/master/app/views/content_items/detailed_guide.html.erb#L23

:100: for switching to component